puppet/modules/jenkins_job_builder/files/centos.org/pipelines/test/nightlyTest.groovy

```diff
@@ -0,0 +1,41 @@
+pipeline {
+    agent { label 'foreman' }
+
+    stages {
+        stage('Setup Environment') {
+            steps {
+                deleteDir()
+                git url: 'https://github.com/theforeman/forklift.git'
+            }
+        }
+        stage('Provision Node') {
+            steps {
+                provision()
+            }
+        }
+        stage('Install Pipeline Requirements') {
+            steps {
+                runPlaybook('playbooks/setup_forklift.yml', cico_inventory('./'), ['@containers/vars/remote.yml'], ['-b'])
+            }
+        }
+        stage('Run Pipeline') {
+            steps {
+                duffy_ssh("cd forklift && ansible-playbook pipelines/pipeline_katello_nightly.yml -e forklift_state=up", 'duffy_box', './')
+            }
+        }
+    }
+
+    post {
+        always {
+            script {
+                runPlaybook('playbooks/get_debug_files.yml', cico_inventory('./'), ['job_name=foreman-katello-nightly-test'])
```

Comment:
Is this playbook at this path?

Reply:
It's not but I had been copying it over to the playbooks for each of use. As it's harder to tell it the giant path to the ansible folder. I think it'd probably be better to change it though.